index.html

```diff
@@ -0,0 +1,82 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <link rel =" stylesheet" href="Style.css" />
+  <title>My CV</title>
+</head>
+<body class="body_class">
+  <header>
+    <h1 class="h1_class">CV</h1>
```

Comment:
It is not necessary to assign class every time, you need to style something. 
`h1 { ... }` would be ok in this case,
Please, name classes according to their sense. "h1" doesn't carry any sense, but something like "heading does"